cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -253,8 +275,8 @@ newtype_of_reg!(
     Xmm,
     WritableXmm,
     OptionWritableXmm,
-    XmmMem,
-    XmmMemImm,
+    reg_mem: (XmmMem, XmmMemAligned aligned:true),
+    reg_mem_imm: (XmmMemImm, XmmMemAlignedImm),
```

Comment:
No `aligned:true` on the `imm` variant?

Reply:
I... don't know how the macro previously compiled. It refers to the `$aligned` variable which was in an entirely different repetition clause and somehow worked? Anyway added!